Dockerfile

```diff
@@ -2,13 +2,14 @@ FROM python:3.13
 
 WORKDIR /code
 
-COPY ./pyproject.toml ./requirements.txt ./
-COPY ./src ./src
+COPY pyproject.toml ./alembic.ini ./
+COPY src ./src
+COPY alembic ./alembic
 
 RUN pip install --upgrade pip
 RUN pip install .
 
 
 EXPOSE 80
 
-CMD ["uvicorn", "src.main:app","--host","0.0.0.0", "--port", "80"]
+CMD ["sh", "-c", "alembic upgrade head && uvicorn src.main:app --host 0.0.0.0 --port 80"]
```

Comment:
Could you elaborate on this command? Why run it in shell? FastAPI also recommend to run it in exec form, read more about it here
https://fastapi.tiangolo.com/deployment/docker/#use-cmd-exec-form